CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v5.1.0
+- This release introduces the **backdrop** option (`true` by default). If set to `false`, the spinner background will remain clickable, without any background color.
```

Comment:
On the "If set to `false`, the spinner background..." can we add "If set to `false`, the spinner background **elements** will remain clickable..."? To emphasize that it's not the spinner itself that is clickable but anything in the background

Reply:
Fair point, fixed !